tests/test_extra_routes.py

```diff
@@ -32,12 +32,12 @@ def delete_item(item_id: str, item: Item):
 
 @app.head("/items/{item_id}")
 def head_item(item_id: str):
-    return JSONResponse(headers={"x-fastapi-item-id": item_id})
+    return JSONResponse(None, headers={"x-fastapi-item-id": item_id})
```

Comment:
If there is not body, it shouldn't be a JSONResponse:

```suggestion
    return Response(headers={"x-fastapi-item-id": item_id})
```

Reply:
I'm not changing the behavior of the test.